components/map/layers/marker.md

```diff
@@ -0,0 +1,234 @@
+    {
+        new MarkerModel()
+        {
+            LatLng = new double[] { 37.7749, -122.4194 },
+            Title = "San Francisco, CA"
+        }
+    };
+
+    public class MarkerModel
+    {
+        public double[] LatLng { get; set; }
+        public string Title { get; set; }
+    }
+}
+````
+
+>caption The result from the above code snippet.
+
+![](../images/marker-layer.png)
+
+## Marker Shapes
+
+The Markers in Map are two types - **Pin** and **PinTarget**. They can be defined using the `Shape` parameter of the `MapLayer` tag. The default visual appearance of the Marker is the **PinTarget**.
+
+>caption Different Marker Shapes.
+
+![](../images/marker-shapes.png)
+
+>caption The example for the above result.
+
+````CSHTML
+@* This code snippet showcases an example of the different Marker Shapes. *@
+
+<TelerikMap Center="@Center" Zoom="3">
+    <MapLayers>
+        <MapLayer Type="@MapLayersType.Marker"
+                  Data="@MarkerData1"
+                  LocationField="@nameof(MarkerModel.LatLng)"
+                  TitleField="@nameof(MarkerModel.Title)" Shape="MapMarkersShape.Pin">
+        </MapLayer>
+
+        <MapLayer Type="@MapLayersType.Tile"
+                  Attribution="@Attribution"
+                  Subdomains="@Subdomains"
+                  UrlTemplate="@UrlTemplate">
+        </MapLayer>
+
+        <MapLayer Type="@MapLayersType.Marker"
+                  Data="@MarkerData2"
+                  LocationField="@nameof(MarkerModel.LatLng)"
+                  TitleField="@nameof(MarkerModel.Title)" Shape="MapMarkersShape.PinTarget">
+        </MapLayer>
+    </MapLayers>
+</TelerikMap>
+
+@code {
+    public string[] Subdomains { get; set; } = new string[] { "a", "b", "c" };
+    public string UrlTemplate { get; set; } = "https://#= subdomain #.tile.openstreetmap.org/#= zoom #/#= x #/#= y #.png";
+    public string Attribution { get; set; } = "&copy; <a href='https://osm.org/copyright'>OpenStreetMap contributors</a>";
+    public double[] Center { get; set; } = new double[] { 30.268107, -97.744821 };
+
+    public List<MarkerModel> MarkerData1 { get; set; } = new List<MarkerModel>()
+    {
+        new MarkerModel()
+        {
+            LatLng = new double[] { 30.268107, -97.744821 },
+            Title = "Austin, TX"
+        }
+    };
+
+    public List<MarkerModel> MarkerData2 { get; set; } = new List<MarkerModel>()
+    {
+        new MarkerModel()
+        {
+            LatLng = new double[] { 37.7749, -122.4194 },
+            Title = "San Francisco, CA"
+        }
+    };
+
+    public class MarkerModel
+    {
+        public double[] LatLng { get; set; }
+        public string Title { get; set; }
+    }
+}
+````
+
+## Marker Tooltip Settings
+
+The `MapLayerMarkerSettingsTooltip` tag allows you to fine tune the tooltips content, appearance and options. You can add HTML, use templates or load an entire page in the tooltip. You can then configure animations, show and hide delays, etc.
```

Comment:
> You can then configure animations, show and hide delays, etc.

This is not currently possible, so it can be removed. The tooltip tag is necessary for exposing the template for now.